client/src/components/Event.test.tsx

```diff
@@ -0,0 +1,42 @@
+
+import { render, screen } from "@testing-library/react";
+import { Event, DateRecord } from "./Event";
```

Comment:
kdyz to otevru, TS hned hlasi chybu `TS1484: DateRecord is a type and must be imported using a type-only import when verbatimModuleSyntax is enabled.` Pro opravu je to treba zmenit:

```
import { Event, type DateRecord } from "./Event";
```